app/models/health/patient.rb

```diff
@@ -917,7 +917,9 @@ def name
 
     def build_team_member!(team_member_class, team_member_user_id, current_user)
       user = User.find(team_member_user_id)
-      team_member = team_member_class.where(patient_id: id, email: user.email).first_or_initialize
+      team_member = team_member_class.where(at[:email].lower.eq(user.email.downcase).to_sql).
```

Comment:
where does `at` get defined?  maybe you mean `team_member_class.arel_table`?

Reply:
Yes, thank you! Never throw in one last tweak after testing.